README.md

```diff
@@ -248,50 +252,55 @@ While [Make](https://www.gnu.org/software/make/) is currently the _de facto_ sta
+While [Task](https://taskfile.dev/) is simpler and easier to use
+than [Make](https://www.gnu.org/software/make/) it still has similar problems:
 
-- Requires to learn Task's YAML structure and the [minimalistic, cross-platform interpreter](https://github.com/mvdan/sh#gosh) which it uses.
+- Requires to learn Task's YAML structure and
+  the [minimalistic, cross-platform interpreter](https://github.com/mvdan/sh#gosh) which it uses.
 - Debugging and testing tasks is not fun.
-- Harder to make some reusable tasks.
-- Requires to "install" the tool. **goyek** leverages `go run` and Go Modules so that you can be sure that everyone uses the same version of **goyek**.
+- Hard to make reusable tasks.
+- Requires to "install" the tool.
 
 ### Bazel
 
-[Bazel](https://bazel.build/) is a very sophisticated tool which is [created to efficiently handle complex and long-running build pipelines](https://en.wikipedia.org/wiki/Bazel_(software)#Rationale). It requires the build target inputs and outputs to be fully specified.
+[Bazel](https://bazel.build/) is a very sophisticated tool which is
+[created to efficiently handle complex and long-running build pipelines](https://en.wikipedia.org/wiki/Bazel_(software)#Rationale).
+It requires the build target inputs and outputs to be fully specified.
 
-**goyek** is just a simple library that is mainly supposed to create a build pipeline consisting of commands like `go vet`, `go test`, `go build`. However, take notice that **goyek** is a library. Nothing prevents you from, for example, using [Mage's target package](https://pkg.go.dev/github.com/magefile/mage/target) to make your build pipeline more efficient.
+**goyek** is just a simple library that is mainly supposed to create a build pipeline
+consisting of commands like `go vet`, `go test`, `go build`.
+However, take notice that **goyek** is a library. Nothing prevents you from,
+for example, using [Mage's target package](https://pkg.go.dev/github.com/magefile/mage/target)
+to make your build pipeline more efficient.
 
 ## Presentations
 
 | Date       | Presentation                                                                            | Description                |
 | ---------- | --------------------------------------------------------------------------------------- | -------------------------- |
-| 2021-03-10 | [taskflow - Create build pipelines in Go](https://github.com/pellared/taskflow-example) | **taskflow** v0.1.1 demo |
+| 2021-05-05 | [goyek - Create build pipelines in Go](https://github.com/pellared/goyek-demo)          | **goyek** v0.3.0 demo      |
+| 2021-03-10 | [taskflow - Create build pipelines in Go](https://github.com/pellared/taskflow-example) | **taskflow** v0.1.1 demo   |
 | 2020-12-14 | [Build pipeline for a Go project](https://github.com/pellared/go-build-pipeline-demo)   | build pipeline using [Make](https://www.gnu.org/software/make/), [Mage](https://github.com/magefile/mage), and **taskflow** v0.1.0 |
```

Comment:
Maybe instead of having this list here it would be better just give a hyperlink to the latest video + google docs (like [here](https://github.com/pellared/taskflow-example)?

Then it could be placed higher in README.

Reply:
I think we can keep this table and one of us can record a dedicated 5min demo like this https://www.youtube.com/watch?v=GOqbD0lF-iA

Reply:
How about collecting these demos/talks in a Wiki page of the project?

Reply:
It may be an optimal option. Let's have a discussion first https://github.com/goyek/goyek/discussions/58